--- lapce/cargo/vendor/alacritty_terminal-0.17.0/src/tty/unix.rs	2006-07-24 03:21:28.000000000 +0200
+++ lapce/cargo/vendor/alacritty_terminal-0.17.0/src/tty/unix.rs	2022-10-24 22:08:12.508524386 +0200
@@ -196,5 +196,6 @@ pub fn new(config: &PtyConfig, window_si
             }
 
-            set_controlling_terminal(slave);
+            // Disable for flatpak-spawn
+            // set_controlling_terminal(slave);
 
             // No longer need slave/master fds.
